### ISSUE TITLE
Printing status is updated only after 1%

### ISSUE THREAD

**Issue by VoliKoN** [cmroche/OctoPrint-HomeAssistant#1]
First of all, great plugin!
Definitely better than the [official integration](https://www.home-assistant.io/integrations/octoprint/).

`binary_sensor.octoprint_printing` is only updating after progress is more than 1%.
This can introduce a delay to an automation on the start of the print.
Maybe you can make it event driven? Maybe `event/PrintStarted` could help?

Offending line:
[`octoprint_homeassistant/__init__.py#L152`](https://github.com/cmroche/OctoPrint-HomeAssistant/blob/master/octoprint_homeassistant/__init__.py#L152)

**Comment by cmroche** [comment]
Duplicate of #7
